3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_mop_up.psql

```diff
@@ -3,28 +3,50 @@
 BEGIN;
 
 --
--- ORIGINAL VIEWS
+-- MODIFIED VIEWS
+-- TODO: Remove these restrictions
 --
 
-CREATE MATERIALIZED VIEW IF NOT EXISTS zambia_irs_structures
+CREATE VIEW pending.restricted_plans AS
```

Comment:
These restrict to a single plan and the Zambia (2020) jurisdiction tree - we can remove once we're happy with that plan's result?